dbhydra/dbhydra_core.py

```diff
@@ -467,6 +467,10 @@ class Jsonable(str):
 
 
 class Mysqldb(AbstractDB):
+    def __init__(self, config_file="config.ini", db_details=None, global_: bool = False, *args, **kwargs):
+        super().__init__(config_file, db_details)
+        if global_:
```

Comment:
this naming is ugly - do you have a better name?

Reply:
moreover - adding parameters to the initialization of db should have a really key reason